services/prometheus/prometheus.go

```diff
@@ -132,6 +134,24 @@ func (svc *Service) reload() error {
 	return errors.Errorf("%d: %s", resp.StatusCode, b)
 }
 
+func (svc *Service) loadBaseConfig() *config.Config {
+	buf, err := ioutil.ReadFile(svc.baseConfigPath)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			svc.l.Errorf("Failed to load base prometheus config %s: %s", svc.baseConfigPath, err)
+		}
+		return new(config.Config)
+	}
+
+	cfg := &config.Config{}
```

Comment:
https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style
https://dave.cheney.net/practical-go/presentations/qcon-china.html#_make_the_zero_value_useful
But please read the whole article.

Reply:
Done